packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableDetailsSteps.tsx

```diff
@@ -174,13 +202,20 @@ export const TransactionsTableDetailsSteps = ({
       {/* The final step, showing the destination chain */}
       <Step
         done={isTxCompleted(tx)}
-        text={`Funds arrived on ${destNetworkName}`}
+        failure={isTxExpired(tx) || isDestinationChainFailure}
+        text={
```

Comment:
Just like we did for Claim, shouldn't the action Redeem/Retry also be a part of the details modal?

<img width="400" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/7558499/4dae5ab5-40ae-4a8e-aa79-9c647068a8e2">

<img width="400" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/7558499/87572626-c954-4064-b54f-b66279df8d05">

Reply:
done